expr/helper/helper.go

```diff
@@ -179,22 +178,33 @@ func SummarizeValues(f string, values []float64) (float64, bool) {
 				rv = av
 			}
 		}
-	case "last":
+	case "last", "current":
 		if len(values) > 0 {
 			rv = values[len(values)-1]
 		}
-
+	case "count":
+		rv = float64(len(values))
+	case "median":
+		val, absent := Percentile(values, 50, true)
+		return val, absent, nil
 	default:
-		f = strings.Split(f, "p")[1]
-		percent, err := strconv.ParseFloat(f, 64)
-		if err == nil {
-			return Percentile(values, percent, true)
+		looks_like_percentile, err := regexp.MatchString(`^p\d`, f)
```

Comment:
Did you prefer this instead of the full test because you think we could support float percentiles? Let's only support integer percentiles, but not `p100`. What do you think? Then you can do a full check with `^p\d\d?$`.

This will still give a weird error when given something like "p2m".

Reply:
Yes, I discovered this while running tests, see : https://github.com/bookingcom/carbonapi/blob/avereha/summarize/expr/functions/summarize/function_test.go#L128
Not sure what to say, if we have a test for this maybe somebody is using it.
Between breaking functionality that someone is using and "weird error" in very extreme cases, I would choose the second one.

Reply:
In the end, we can search the logs and see if anyone is using fractional percentiles. 
I agree that they don't seem very useful, so I'm going to implement you suggestion.

Reply:
Done.